client/layout/universal-navbar-footer/index.tsx

```diff
@@ -209,68 +213,64 @@ const UniversalNavbarFooter = () => {
 									className="lp-language-picker__content"
 									title={ translate( 'Change Language' ) }
 									onChange={ ( e ) => navigate( e.target.value ) }
```

Comment:
This doesn't seem to work properly as it needs a refresh for the text to be translated.
I'd suggest adding a `window.location.reload()` after `navigate`. I tested and it shouldn't affect SSR as this never gets called on the server.

Reply:
Thanks @claudiucelfilip  - I have updated this now!